cfgov/jinja2/v1/_includes/molecules/related-posts.html

```diff
@@ -86,7 +86,7 @@ <h4 class="u-mb5">
             {{ post.text | safe }}
         </p>
         {% endif %}
-        <p class="date">
+        <p class="date u-mt0">
```

Comment:
This shouldn't be needed once the [fix in cf-typography](https://github.com/cfpb/cf-typography/pull/45/files#diff-0e4058fa92e42934a387c828c10dce71R240) is merged.